<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>419</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>437</width>
    <height>419</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="main">
       <attribute name="title">
        <string>Main</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>20</y>
             <width>171</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>You are using this layout :</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>80</y>
             <width>171</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>hotkeys status :</string>
           </property>
          </widget>
          <widget class="QWidget" name="horizontalLayoutWidget">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>40</y>
             <width>321</width>
             <height>51</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="layout_filename_label">
              <property name="text">
               <string>&lt;layout_filename&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="layout_settings_button">
              <property name="styleSheet">
               <string notr="true">QPushButton {
    border: none;
    background: transparent;
    text-align: left;
    color: blue;
}
QPushButton:hover {
    color: blue;
}
QPushButton:pressed {
    color: blue;
}</string>
              </property>
              <property name="text">
               <string>Layout Settings</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="verticalLayoutWidget">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>110</y>
             <width>321</width>
             <height>31</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="hotkey_status_label">
              <property name="text">
               <string>&lt;hotkeys_status&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="edit_hotkeys_button">
              <property name="styleSheet">
               <string notr="true">QPushButton {
    border: none;
    background: transparent;
    text-align: left;
    color: blue;
}
QPushButton:hover {
    color: blue;
}
QPushButton:pressed {
    color: blue;
}</string>
              </property>
              <property name="text">
               <string>Edit Hotkeys</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="maximumSize">
           <size>
            <width>419</width>
            <height>177</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <widget class="QPushButton" name="pushButton">
           <property name="geometry">
            <rect>
             <x>190</x>
             <y>50</y>
             <width>171</width>
             <height>61</height>
            </rect>
           </property>
           <property name="text">
            <string>Start!</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layout">
       <attribute name="title">
        <string>Layout</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>150</y>
          <width>181</width>
          <height>131</height>
         </rect>
        </property>
        <property name="title">
         <string>Grid</string>
        </property>
        <widget class="QPushButton" name="visualize_grid">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>30</y>
           <width>121</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Visualize Grid</string>
         </property>
        </widget>
        <widget class="QPushButton" name="add_table_slot_button">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>90</y>
           <width>21</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
        <widget class="QPushButton" name="reduce_table_slot_button">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>90</y>
           <width>21</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
        <widget class="QWidget" name="horizontalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>71</width>
           <height>21</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Total Slots:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="total_tables_label">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="hotkeys">
       <attribute name="title">
        <string>Hotkeys</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="table_setup">
       <attribute name="title">
        <string>Table Setup</string>
       </attribute>
       <widget class="QWidget" name="horizontalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>170</y>
          <width>117</width>
          <height>73</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Height :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="table_height">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_4">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>170</y>
          <width>114</width>
          <height>73</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Width :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="table_width">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_5">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>70</y>
          <width>371</width>
          <height>71</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Table String :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="table_string_search"/>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="table_setup_search">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>250</y>
          <width>101</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
       <widget class="QPushButton" name="table_setup_save">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>250</y>
          <width>101</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>10</y>
          <width>381</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Open desired a single table and input table name then press search</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>30</y>
          <width>341</width>
          <height>41</height>
         </rect>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::PreventContextMenu</enum>
        </property>
        <property name="text">
         <string>if detected, table should be highlighted, as well as height and width to be populated</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>437</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMain">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuMain"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
